setup.cfg

```diff
@@ -28,13 +28,13 @@ python_requires = >=3.7
 zip_safe = False
 install_requires =
     Babel>=2.8
-    Flask-BabelEx>=0.9.4
-    invenio-base>=1.2.11
+    Flask-Babel>=2.0.0
```

Comment:
yes we should require `Flask>=2.2`. I tried with `Flask==2.0.0` and it was not working.

Reply:
I think we should because of the deprecation/future removal of `app.json_encoder` (if I am not mistaken).
For the version, if RDM v9.x works with this (having Flask>=2.2) installed, then we could even release a `minor`: I didn't find [any usage](https://github.com/search?q=org%3Ainveniosoftware+set_locale&type=code) of the `set_locale` func around...
Releasing a major will mean that we will have to pin `Flask<2.3` in RDM 9.x, because it will probably break if not all deprecation warnings are fixed.

Reply:
In my opinion it would be better to not release until all other packages are tested locally. The problem is that flask-babel and flask-babelex are installed both simultaneously with conflicting babel versions.